pkg/metricscollector/prommetrics.go

```diff
@@ -80,53 +90,97 @@ var (
+	scalerErrors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "scaler",
+			Name:      "errors_total",
+			Help:      "The total number of errors encountered for each scaler.",
+		},
+		metricLabels,
+	)
+	scaledObjectErrorsDeprecated = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: DefaultPromMetricsNamespace,
 			Subsystem: "scaled_object",
 			Name:      "errors",
-			Help:      "Number of scaled object errors",
+			Help:      "DEPRECATED - will be removed in 2.15 - use 'scaled_object_errors_total' instead.",
 		},
 		[]string{"namespace", "scaledObject"},
 	)
-
-	triggerTotalsGaugeVec = prometheus.NewGaugeVec(
+	scaledObjectErrors = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "scaled_object",
+			Name:      "errors_total",
+			Help:      "The number of errors that have occurred for each ScaledObject.",
+		},
+		[]string{"namespace", "scaledObject"},
+	)
+	triggerTotalsGaugeVecDeprecated = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: DefaultPromMetricsNamespace,
 			Subsystem: "trigger",
 			Name:      "totals",
+			Help:      "DEPRECATED - will be removed in 2.15 - use 'trigger_handled_total' instead.",
 		},
 		[]string{"type"},
 	)
-
-	crdTotalsGaugeVec = prometheus.NewGaugeVec(
+	triggerHandled = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "trigger",
+			Name:      "handled_total",
+			Help:      "Total number of triggers per trigger type handled.",
+		},
+		[]string{"type"},
+	)
+	crdTotalsGaugeVecDeprecated = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: DefaultPromMetricsNamespace,
 			Subsystem: "resource",
 			Name:      "totals",
+			Help:      "DEPRECATED - will be removed in 2.15 - use 'resource_handled_total' instead.",
 		},
 		[]string{"type", "namespace"},
 	)
-
-	internalLoopLatency = prometheus.NewGaugeVec(
+	crdHandled = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "resource",
+			Name:      "handled_total",
+			Help:      "Total number of KEDA custom resources per namespace for each custom resource type (CRD) handled.",
+		},
+		[]string{"type", "namespace"},
+	)
+	internalLoopLatencyDeprecated = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: DefaultPromMetricsNamespace,
 			Subsystem: "internal_scale_loop",
 			Name:      "latency",
-			Help:      "Internal latency of ScaledObject/ScaledJob loop execution",
+			Help:      "DEPRECATED - will be removed in 2.15 - use 'internal_scale_loop_latency_seconds' instead.",
+		},
+		[]string{"namespace", "type", "resource"},
+	)
+	internalLoopLatency = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: DefaultPromMetricsNamespace,
+			Subsystem: "internal_scale_loop",
+			Name:      "latency_seconds",
```

Comment:
same as above

Reply:
I guess it's been settled that KEDA will use `Seconds` in metrics.